src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -2161,6 +2161,7 @@ constexpr char const CORE_RPC_STATUS_TX_LONG_POLL_MAX_CONNECTIONS[] = "Daemon ma
       std::string status;       // General RPC error code. "OK" means everything looks good.
       uint64_t emission_amount; // Amount of coinbase reward in atomic units.
       uint64_t fee_amount;      // Amount of fees in atomic units.
+      uint64_t burn_amount;      // Amount of burnt loki.
```

Comment:
Need to add a KV_SERIALIZE(burn_amount) below so it gets serialized into the RPC json response.